controller/appcontroller.go

```diff
@@ -183,12 +188,15 @@ func NewApplicationController(
 	enableK8sEvent []string,
 	hydratorEnabled bool,
 ) (*ApplicationController, error) {
-	log.Infof("appResyncPeriod=%v, appHardResyncPeriod=%v, appResyncJitter=%v", appResyncPeriod, appHardResyncPeriod, appResyncJitter)
+	log.Infof("Initializing ApplicationController: appResyncPeriod=%v, appHardResyncPeriod=%v, appResyncJitter=%v", appResyncPeriod, appHardResyncPeriod, appResyncJitter)
+
 	db := db.NewDB(namespace, settingsMgr, kubeClientset)
+	// use default rate limiter config if not provided
```

Comment:
Do you think we really need this comment? It's apparent what the following line does. 
Overall, if we need comments to explain the code, it's better to improve the code and  remove the comments